src/sentry/receivers/outbox/control.py

```diff
@@ -121,15 +120,12 @@ def process_async_webhooks(payload: Mapping[str, Any], region_name: str, **kwds:
 
 
 @receiver(process_control_outbox, sender=OutboxCategory.SEND_SIGNAL)
-def process_send_signal(
-    payload: Mapping[str, Any], shard_identifier: int, shard_scope: OutboxScope, **kwds: Any
-):
-    if shard_scope == OutboxScope.ORGANIZATION_SCOPE:
-        organization_service.send_signal(
-            organization_id=shard_identifier,
-            args=payload["args"],
-            signal=RpcOrganizationSignal(payload["signal"]),
-        )
+def process_send_signal(payload: Mapping[str, Any], shard_identifier: int, **kwds: Any):
```

Comment:
Now that I'm enforcing every category have a unique scope, there is no reason to do this comparison.  I originally thought about a future where this category might be used for multiple different scopes, but I have since changed my mind.